### ISSUE TITLE
Images are missing alt text

### ISSUE THREAD

**Issue by Drafell** [ryceg/Eigengrau-s-Essential-Establishment-Generator#449]
The "Support Me...","Join Our Discord" and "Visit Our Github" link images need to have associated alt text. This can be the same as the text on the images. This will also fix a problem with the parent <a> tags not actually having link text, as the alt text in the image will fulfill this need.
